internal/namespaces/instance/v1/command_server_create.go

```diff
@@ -431,17 +454,20 @@ func validateLocalVolumeSizes(api *instance.API, volumes []*instance.VolumeTempl
 	}
 
 	// Validate total size.
-	var vConstr *instance.ServerTypeVolumeConstraintSizes
-	if st, exists := serverTypesRes.Servers[commercialType]; exists {
-		vConstr = st.VolumesConstraint
+	if serverType, exists := serverTypesRes.Servers[commercialType]; exists {
+		volumeConstraint := serverType.VolumesConstraint
 
-		if localVolumeTotalSize < vConstr.MinSize || localVolumeTotalSize > vConstr.MaxSize {
-			min := humanize.Bytes(uint64(vConstr.MinSize))
-			if vConstr.MinSize == vConstr.MaxSize {
+		if rootVolume, exist := volumes["0"]; !exist || rootVolume == nil {
```

Comment:
Maybe add a comment like:
`// If no root volume provided, count the default root volume size added by the API.`